src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -154,6 +154,12 @@ $.BootstrapTable = class extends $.BootstrapTable {
     super.init()
   }
 
+  load (data) {
+    super.load(data)
+
+    UtilsFilterControl.createControls(this, UtilsFilterControl.getControlContainer(this))
```

Comment:
Should we add an if statement?

```js
if (!this.options.filterControl) {
  return;
}
```

Reply:
yes we should, thanks for the hint.
Added in the last commit.